package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-slingshot",
-  "version": "5.0.0",
+  "version": "7.0.0",
```

Comment:
Well embarrassingly, when I announced 6.0, I didn't update package.json to reflect it. I would consider the config move a breaking change, so that's why I went to 7. Make sense?